utils/spacewalk-common-channels.ini

```diff
@@ -4058,6 +4058,89 @@ checksum = sha256
 base_channels = debian-12-pool-amd64-uyuni
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/Debian12-Uyuni-Client-Tools/Debian_12/
 
+[fuss-12-pool-amd64-uyuni]
+label    = fuss-12-pool-amd64-uyuni
+checksum = sha256
+archs    = amd64-deb
+repo_type = deb
+name     = FUSS 12 (bookworm) pool for amd64 for Uyuni
+gpgkey_url =
+gpgkey_id =
+gpgkey_fingerprint =
+repo_url = http://deb.debian.org/debian/dists/bookworm/main/binary-amd64/
+
+[fuss-12-amd64-main-updates-uyuni]
+label    = fuss-12-amd64-main-updates-uyuni
+name     = FUSS 12 (bookworm) AMD64 Main Updates for Uyuni
+archs    = amd64-deb
+repo_type = deb
+checksum = sha256
+base_channels = fuss-12-pool-amd64-uyuni
+repo_url = http://deb.debian.org/debian/dists/bookworm-updates/main/binary-amd64/
+
+[fuss-12-amd64-main-security-uyuni]
+label    = fuss-12-amd64-main-security-uyuni
+name     = FUSS 12 (bookworm) AMD64 Main Security for Uyuni
+archs    = amd64-deb
+repo_type = deb
+checksum = sha256
+base_channels = fuss-12-pool-amd64-uyuni
+repo_url = http://deb.debian.org/debian-security/dists/bookworm-security/updates/main/binary-amd64/
+
+[fuss-12-amd64-main-backports-uyuni]
+label    = fuss-12-amd64-main-backports-uyuni
+name     = FUSS 12 (bookworm) AMD64 Main Backports for Uyuni
+archs    = amd64-deb
+repo_type = deb
+checksum = sha256
+base_channels = fuss-12-pool-amd64-uyuni
+repo_url = http://deb.debian.org/debian/dists/bookworm-backports/main/binary-amd64/
+
+[fuss-12-custom-main-amd64-uyuni]
```

Comment:
Do they call it custom on their side?

Reply:
I did not know how to call it, otherwise the name was the same than the base channel. Open to suggestions...

Reply:
What's the name of the file that has it? Or what does the doc says when talking about this repo?

Reply:
> What's the name of the file that has it? Or what does the doc says when talking about this repo?

```
root@fuss-12:~# cat /etc/apt/sources.list
deb http://deb.debian.org/debian/ bookworm main non-free-firmware contrib non-free
deb-src http://deb.debian.org/debian/ bookworm main non-free-firmware contrib non-free

deb http://security.debian.org/debian-security bookworm-security main non-free-firmware contrib non-free
deb-src http://security.debian.org/debian-security bookworm-security main non-free-firmware contrib non-free

root@fuss-12:~# cat /etc/apt/sources.list.d/
archive_fuss_bz_it.list  google-chrome.list       

root@fuss-12:~# cat /etc/apt/sources.list.d/archive_fuss_bz_it.list /etc/apt/sources.list.d/google-chrome.list 
deb [signed-by=/usr/share/keyrings/fuss-keyring.gpg] http://archive.fuss.bz.it/ bookworm main contrib non-free
### THIS FILE IS AUTOMATICALLY CONFIGURED ###
# You may comment out this entry, but any other modifications may be lost.
deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main
```

Do you want to change `fuss-12-custom-main-amd64-uyuni` to `fuss-12-fuss-main-amd64-uyuni`?
Second question, as it seems to have Google Chrome repository enabled by default, should I add it as well?

Reply:
Now I'm wondering whether the repos are completely accurate or not (even if everything's working), probably being very picky, but according to that, I should have:
- 4 repos for the first line (main, non-free-firmware, contrib, non-free)
- 4 repos for the 3rd line (same ones, but in debian-security)
- 3 repos for the FUSS specific (main, contrib, non-free)
- The 2 usual ones for Uyuni tools (stable + development).
- Google Chrome repository, if we decide to add it.

Thoughts?

Reply:
Mmm, I only see you added 9 repos, two of them being client tools... but in this list you have 14?

So looks to me you're indeed missing repositories, even from Debian maybe?

I'd add all.

My suggestion would be:
- Those coming from Debian, to start with `fuss-12-debian-`. They are not really custom, right? Just Debian12 repos :-)
- Those coming from FUSS, to start with `fuss-12-fuss-`. They are really coming from FUSS
- Those for the Uyuni tools, keep it as is
- The one for Chrome: `fuss-12-googlechrome-amd64-uyuni`. It comes from googlechrome.

Keep in mind changing the prefix for Debian means you will need to refresh the the bootstrap repo definition as well.